src/dealDashboard/discussionThread/singleComment/singleComment.ts

```diff
@@ -0,0 +1,100 @@
+import { EthereumService } from "services/EthereumService";
+import { autoinject, bindable, computedFrom } from "aurelia-framework";
+import { Router } from "aurelia-router";
+import { IComment, IProfile } from "entities/DealDiscussions";
+import { DateService } from "services/DateService";
+import "./singleComment.scss";
+
+@autoinject
+export class SingleComment {
+  @bindable private comment: IComment;
+  @bindable private repliesTo: IComment;
+  @bindable private repliesToProfile: IProfile;
+  @bindable private author: string;
+  @bindable private profile: IProfile;
+  @bindable private loading: string;
+  @bindable private highlighted: number;
+  @bindable private index: number;
+  @bindable private isReply?: boolean = false;
+  @bindable callback;
```

Comment:
(discussed)

```suggestion
  @bindable commentAction;
```

Reply:
`callback` is necessary- otherwise it doesn't work...

Reply:
I was only suggesting to rename.
But only nitpick here, so can be ignored.

Reply:
agreed, "callback" means nothing.  Better to have helpful names.